test/c_api/database_test.cpp

```diff
@@ -2,11 +2,10 @@
 
 class CApiDatabaseTest : public EmptyDBTest {
 public:
-    void SetUp() override {}
+    void SetUp() override { EmptyDBTest::SetUp(); }
```

Comment:
@mewim Is it ok to add `EmptyDBTest::SetUp()` here? I basically need to set the temporary dynamic database path instead of using the fixed one.

Reply:
Sorry I missed this. Since it is only setting a path it is fine.